pkg/users/account-details.js

```diff
@@ -164,16 +163,22 @@ export function AccountDetails({ accounts, groups, current_user, user, shells })
                 .catch(show_unexpected_error);
     }
 
-    if (!accounts.length) {
+    // Whenever details change we can clear the locked flag
+    useEffect(
+        () => {
+            setEditedLocked(null);
+        },
+        [details],
+    );
+
+    if (isLoading) {
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-21865-e74b7696-20250721-143844-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)